install/terraform/modules/aks/aks.tf

```diff
@@ -60,7 +60,7 @@ resource "azurerm_kubernetes_cluster" "agones" {
   resource_group_name = azurerm_resource_group.agones_rg.name
   dns_prefix          = "agones"
 
-  kubernetes_version = "1.15.10"
+  kubernetes_version = "1.16.10"
```

Comment:
Is 1.16.10 the latest version of 1.16 on aks?

Reply:
According to the docs, yes: https://docs.microsoft.com/en-us/azure/aks/supported-kubernetes-versions
I am going to renew my subscription to AKS and make a fast test.
Also this command would be useful:
```
az aks get-versions --location eastus --output table
```

Reply:
Renewed my subscription. Testing EKS cluster with Terraform now.
```
az aks get-versions --location eastus --output table
KubernetesVersion    Upgrades
-------------------  -------------------------
1.17.9               None available
1.17.7               1.17.9
1.16.13              1.17.7, 1.17.9
```

Reply:
Thanks for double checking.